### DISCUSSION TITLE
when will 1.18 support be added

### DISCUSSION THREAD

**Discussion by lordi-erez** [gdavid04/phi#14] [General]
this mod makes psi so much better w/ its flow control, i was wondering if 1.18 support is planned now that psi has been updated

**Comment by gdavid04** [comment]
For now I'll be focusing on the anniversary update which will be released for 1.16. I also want to finish a PR I have been working on for a while for Psi. After that I plan to port features to 1.18 gradually as my time allows. A reason I want to wait until I finish said PR is that some currently unreleased Phi features would be significantly easier to implement correctly with the changes I'm working on.

**Reply by gdavid04** [reply]
Update: 1.19 port is currently work in progress, perhaps I'll also consider an 1.18 port depending on various factors such as what time allows, how difficult it would be and whether Psi gets ported to a newer version in the meantime.
